src/daemon/dlt-daemon.c

```diff
@@ -1287,13 +1305,8 @@ void dlt_daemon_local_cleanup(DltDaemon *daemon, DltDaemonLocal *daemon_local, i
     /* Ignore result */
     dlt_file_free(&(daemon_local->file), daemon_local->flags.vflag);
 
-#ifndef DLT_USE_UNIX_SOCKET_IPC
```

Comment:
This change should go to separate commit, as you are reverting [https://github.com/GENIVI/dlt-daemon/commit/7d9d156fb8571aff4eb89db201a26af0d0370dc7](url).

Reply:
I cannot find this commit anymore. Was there a force-push?

Reply:
This is still available. If you cannot jump from the link, please try to find "7d9d156 dlt-daemon: unlink application socket" from the master branch.